nb

```diff
@@ -440,8 +440,15 @@ _option_value_is_present() {
 #
 # Default: `$HOME/.nbrc`
 #
-# The location of the .nbrc configuration file.
-export NBRC_PATH="${NBRC_PATH:-"${HOME}/.${_ME}rc"}"
+# Check if XDG_CONFIG_HOME exists
+if [[ -n "${XDG_CONFIG_HOME}" ]]; then
```

Comment:
Currently, I think this would always use `XDG_CONFIG_HOME/nb/.nbrc` if `XDG_CONFIG_HOME` is defined. It should probably still default to using `$HOME/.nbrc` if the file already exists so it doesn't break existing installations and setups.